### DISCUSSION TITLE
Disable the autofix which introduces line breaks

### DISCUSSION THREAD

**Discussion by halukakin** [WeblateOrg/weblate#9070] [Q&A] [answered]
How can I disable this autofix which introduces line breaks to my yaml files?
I understand it imposes some sort of line length limit.
The wrap happens around 80-90 chars.

![Screen Shot 2023-04-10 at 05 30 52](https://user-images.githubusercontent.com/22102099/230997765-727c14a7-1638-4c60-ba81-e63ba944f15f.png)

**Comment by nijel** [comment, marked as answer]
You can customize YAML formatting via add-on, see https://docs.weblate.org/en/latest/admin/addons.html#customize-yaml-output

**Reply by halukakin** [reply]
Worked great. Thank you!
